crates/router/src/routes/payments.rs

```diff
@@ -205,6 +266,21 @@ pub async fn payments_confirm(
     .await
 }
 
+/// Payments - Capture
+///
+/// To capture payments for a currently uncaptured Payment with its status as requires_capture
```

Comment:
```suggestion
/// To capture payments for a currently uncaptured payment
```

@bernard-eugine What do you think, do we need the `with its status as requires_capture` part again?